docs/CODE_OF_CONDUCT-km.md

```diff
@@ -0,0 +1,47 @@
+# ក្រមសីលធម៌របស់អ្នកចូលរួម
+
+ក្នុងនាមជាអ្នករួមចំណែក និងអ្នកថែរក្សាគម្រោងនេះ និងក្នុងគោលបំណងជំរុញសហគមន៍បើកចំហ និងស្វាគមន៍ 
+យើងសន្យាគោរពមនុស្សទាំងអស់ដែលរួមចំណែកតាមរយៈការរាយការណ៍បញ្ហា ការបង្ហោះសំណើលក្ខណៈពិសេស 
+ការធ្វើបច្ចុប្បន្នភាពឯកសារ ការដាក់សំណើរទាញ ឬបំណះ និងសកម្មភាពផ្សេងទៀត។
+
+យើងប្តេជ្ញាធ្វើឱ្យការចូលរួមនៅក្នុងគម្រោងនេះដោយគ្មានការយាយី
+បទពិសោធន៍សម្រាប់មនុស្សគ្រប់គ្នា ដោយមិនគិតពីកម្រិតនៃបទពិសោធន៍ ភេទ ភេទ
+អត្តសញ្ញាណ និងការបញ្ចេញមតិ ទំនោរផ្លូវភេទ ពិការភាព រូបរាងផ្ទាល់ខ្លួន
```

Comment:
`ភេទ អត្តសញ្ញាណ` It should `អត្តសញ្ញាណភេទ`. 
`Gender Identity` we don't this word in Khmer. But we can use word command translation with no space to indicated that it is one word.